test/integrations/routes/articleRating.test.js

```diff
@@ -0,0 +1,34 @@
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../../index';
+
+chai.use(chaiHttp);
+
+describe('Article Rating API endpoint', () => {
+  const token = process.env.JWT_TOKEN;
+  describe('/rating Post Endpoint', () => {
+    const rateArticle = {
+      stars: 5,
+    };
+
+    it('should create an article', async () => {
```

Comment:
#### Nitpick
- Are you creating an article or creating an  article then adding a rating.

Reply:
@aanchirinah I'm only adding a rating to an existing article, I will change the description to something fitting.